### PR TITLE
Update README.md 

### PR DESCRIPTION
**Pull request by yiwen101**
- [ ] I have updated the documentation

### CODE REVIEW

**Review by martin-henz — APPROVED** [review]
Thanks for sharing.

File: README.md [hunk 1]
```diff
@@ -153,6 +153,7 @@ The regression tests are generated using `jest` and stored as snapshots in `src/
 $ yarn test --updateSnapshot
 ```
 and then typing `a` to update all snapshots.
+（if you run this on a macBook with ARM chip, you may run into errors; try to do this on a computer with x86 chip instead)
```
Comment (RichDom2185) [inline README.md:156]: May I know what errors, exactly? I'm on an M1 Macbook and there have been no errors whenever I try to update snapshots.

Comment (yiwen101) [reply README.md:156]: there is not "node-canvas" version available for my Mac. On visiting the website, I also did not find available version.

Comment (RichDom2185) [reply README.md:156]: What's your node version? As well as what Mac are you using?

**Pull request merged**